docs/index.md

```diff
@@ -142,6 +147,29 @@ The URL at `<<iss value from Signed JWT>>` SHALL NOT include a trailing `/`. For
 }
 ```
 
+### Certificates
+
+X.509 certificates can be used by issuers to indicate the issuer's participation in a PKI-based trust framework.
+
+If the Verifier supports PKI-based trust frameworks and the Health Card issuer includes the `"x5c"` parameter in matching JWK entries from the `.keys[]` array,
+the Verifier establishes that the issuer is trusted as follows:
+1. Verifier validates the leaf certificate's binding to the Health Card issuer by:
+    * matching the `<<iss value from Signed JWT>>` to the value
+of a `uniformResourceIdentifier` entry in the certificate's Subject Alternative Name extension
+(see [RFC5280](https://tools.ietf.org/html/rfc5280#section-4.2.1.6)), and
+    * verifying the signature in the Health Card using the public key in the certificate.
+2. Verifier constructs a valid certificate path of unexpired and unrevoked certificates to one of its trusted anchors
+ (see [RFC5280](https://tools.ietf.org/html/rfc5280#section-6)).
+
+
+### Key Management
```

Comment:
Should we add lifetime recommendations? I'd recommend 1 year to match the new norms for endpoint TLS certs.

Reply:
re x5u, I'm not sure there is real benefit over x5c because x5u can still only point one cert or cert chain per the current JWK spec (basically same info as x5c, so no real upside and it requires one additional link dereference.) it's too bad p7b (as allowed in X509 AIA URLs) isn't allowed as an option at x5u endpoint in the JWK and JWS specs. I did add some text in response to an earlier review comment about how multi-framework situations could be handled in a JWKS with multiple JWK entries, though, which hopefully addresses this sufficiently.